### DISCUSSION TITLE
Error while trying to upgrade airflow 2.2.5 to 2.3.3

### DISCUSSION THREAD

**Discussion by DantasB** [apache/airflow#25162] [Q&A]
When i was trying to upgrade my airflow from 2.2.5 to 2.3.3 on kubernetes, i got some errors while running the create-users task. The error is related about the Permission of this declarative base.

![image](https://user-images.githubusercontent.com/15825133/179790597-c9c80e4a-2fbf-4e07-a479-69808762b77e.png)

Any idea about how to fix that? (I'm getting this error when trying to upgrade to 2.3+ but i can downgrade to 2.0, 2.1, etc.

**Comment by potiuk** [comment]
Likely your environment is broken - you might have two different Airlfow versions or FAB versions installed at the same time in different packages. I recommend you install airflow from scratch in completely new virtualenv

**Reply by DantasB** [reply]
This airflow is completly installed from scratch. Empty database and new namespace. 

I have two airflows in the sabe kubernetes cluster, is that still a problem?

**Reply by potiuk** [reply]
No idea - maybe you used the same database./schema for both - maybe even accidenally (wild guessing).

 But this does not happen to others, so you must have done something wrong. But what it is - only you can tell because this does not happen when you install airllow "normally".
